README.md

```diff
@@ -167,10 +169,10 @@ This plugin creates device plugin endpoints based on the configurations given in
 
 
 
-|     Field      | Required |        Description        |                       Type - Accepted values                        |                          Example/Accepted values                          |
+|     Field      | Required |        Description        |                       Type - Accepted values                        |                          Example/Accepted values                                                   |
 |----------------|----------|---------------------------|---------------------------------------------------------------------|---------------------------------------------------------------------------|
-| "resourceName" | Yes      | Endpoint resource name    | `string` - must be unique and should not contain special characters | "sriov_net_A"                                                             |
-| "selectors"    | No       | A map of device selectors | Each selector is a map of  `string` list.                           | "vendors": ["8086"], "devices": ["154c", "10ed"], "drivers": ["vfio-pci"] |
+| "resourceName" | Yes      | Endpoint resource name    | `string` - must be unique and should not contain special characters | "sriov_net_A"                                                                                      |
+| "selectors"    | No       | A map of device selectors | Each selector is a map of  `string` list.                           | "vendors": ["8086"], "devices": ["154c", "10ed"], "drivers": ["vfio-pci"], "ifnames": ["enp2s2f0"] |
 
```

Comment:
Please a description of this selector  at the end of the section "Assumptions".

Reply:
fixed